### DISCUSSION TITLE
Dropdown with Virtual Scroll and Groups

### DISCUSSION THREAD

**Discussion by mnhock** [primefaces/community#79] [PrimeNG] [answered]
Does PrimeNG's Dropdown component support Virtual Scroll activated per group, and if so, how can this functionality be utilized? The documentation does not provide examples for this specific feature.

In the version 13.x we used it per group and it works per group. In the latest version unfortunately it is not working anymore.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

To make it clear could you please explain `Virtual Scroll activated per group`

As I understand from above, unfortunately, it's not supported per group but maybe you can assign the group's height as scrollHeight and it might work. In the new virtual scroll implementation, you need to define a `scrollHeight` and `virtualScrollItemSize`. For example; if your groups have 5 items and each individual item has 30px height, you can assign 150px as scrollHeight, it'll work per group.

We've reworked the virtualscroll in v14. Because of that, some features of v13 might not work as you expected in >= v14.

**Reply by mnhock** [reply]
Thanks for this information! 

Unfortunately, we also have considerable performance problems with the new version regarding this feature, which were not before. We will investigate.
